src/modules/tests/test-data/cat-a-mod1/single-fault-competencies/single-fault-competencies.reducer.ts

```diff
@@ -13,7 +13,9 @@ export function singleFaultCompetenciesReducer(
         ...state,
         [action.competencyName]: action.outcome,
       };
-    case singleFaultCompetencyActions.REMOVE_SINGLE_FAULT_COMPETENCY_OUTCOME:
+    case singleFaultCompetencyActions.REMOVE_SINGLE_FAULT_COMPETENCY_OUTCOME ||
+      singleFaultCompetencyActions.REMOVE_SINGLE_DANGEROUS_FAULT_COMPETENCY_OUTCOME ||
+      singleFaultCompetencyActions.REMOVE_SINGLE_SERIOUS_FAULT_COMPETENCY_OUTCOME:
```

Comment:
there should be a case for each action instead of doing or's

```
case singleFaultCompetencyActions.REMOVE_SINGLE_FAULT_COMPETENCY_OUTCOME: 
case singleFaultCompetencyActions.REMOVE_SINGLE_DANGEROUS_FAULT_COMPETENCY_OUTCOME:
case singleFaultCompetencyActions.REMOVE_SINGLE_SERIOUS_FAULT_COMPETENCY_OUTCOME:
```